pages/common/paste.md

```diff
@@ -0,0 +1,23 @@
+# paste
+
+> Merge lines of files.
+
+- join all the lines into one, using TAB as delimiter to separate the columns.
```

Comment:
I think the "to separate the columns" is unnecessary and possibly even confusing. How about removing that part?